haskell-src/lib/ChainwebData/Env.hs

```diff
@@ -385,8 +446,12 @@ commands = hsubparser
        (progDesc "Gaps Worker - Fills in missing blocks lost during backfill or listen (DEPRECATED)"))
   <> command "single" (info singleP
        (progDesc "Single Worker - Lookup and write the blocks at a given chain/height"))
-  <> command "server" (info (Server <$> serverP)
+  <> command "server" (info (Worker <$> serverP)
        (progDesc "Serve the chainweb-data REST API (also does listen)"))
+  <> command "etl" (info (Worker <$> etlP)
+       (progDesc "ETL Worker - Fills in missing blocks and listens for new ones"))
+  <> command "http" (info (Worker <$> httpP)
+       (progDesc "HTTP Worker - Serves the chainweb-data REST API"))
```

Comment:
I wonder if we should avoid adding two new subcommands. After all, there's very little difference between `listen` and `etl` now. AFAICT, the only difference is that `etl` has the `run-fill` argument, which we could add to `listen` and `etl` has a slightly different logging format.

Similarly `http` is just `server` without the ETL in it, so maybe we should instead add an optional `--no-etl` flag to `server`, or maybe even just make the `--service` and `--p2p` flags optional and treat it as `http` when they're missing?

Either way, I think there's some room for simplification here.